packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -56,6 +73,14 @@ export const Webchat = props => {
       }
       if (devSettings) updateDevSettings(devSettings)
     } catch (e) {}
+    if (appId) {
+      pusher = new Pusher('da85029877df0c827e44')
+      if (!Object.keys(pusher.channels.channels).length) {
+        //TODO: remove my endpoint, generic one with socket
```

Comment:
is this done? otherwise, should we create a ticket to track it?

Reply:
Yes, it's done. I should remove the TODO comment